### DISCUSSION TITLE
Training: "styles" folder does not exists in the latest install.

### DISCUSSION THREAD

**Discussion by LondonWebFactory** [vercel/next.js#39966] [Help]
Greetings!

With regard to this training page:
https://nextjs.org/learn/basics/assets-metadata-css/css-styling

Unfortunately there doesn't seem to be a "styles" folder in the latest install.

Kind Regards,
Adrian Smith

**Comment by adxl** [comment]
It seems that the `styles` folder has been replaced by some inline `<style>` tags in `pages/index.js`

**Reply by LondonWebFactory** [reply]
I just thought the team might like to know so they could update the training docs.

Cheers,
Adrian
